server/routes/static.ts

```diff
@@ -13,6 +13,7 @@ export default function staticRoutes(controllers: Controllers, router: Router, s
   const { staticController } = controllers
 
   get(paths.static.maintenancePage.pattern, staticController.maintenancePage(), {})
+  get(paths.static.privacyNotice.pattern, staticController.privacyNoticePage(), {})
```

Comment:
@tahb @edavey do you think we need an audit event for viewing this privacy page?  Or are the audit logs more for a trail to see what data a user has accessed?

Reply:
no I don't think it's appropriate to record viewing the privacy page in the HMPPS Audit service. 

That service existis to provide an audit log of who has attempted to view what, from a security, authorisation and need-to-view perspective. The Audit service is about understanding whether access to controlled data is being abused or violations to access control are being attempted.